editor/js/ui/gitConfig.js

```diff
@@ -0,0 +1,312 @@
+            //                     .appendTo(header)
+            //                     .click(function(e) {
+            //                         var textarea = document.createElement("textarea");
+            //                         textarea.style.position = 'fixed';
+            //                         textarea.style.top = 0;
+            //                         textarea.style.left = 0;
+            //                         textarea.style.width = '2em';
+            //                         textarea.style.height = '2em';
+            //                         textarea.style.padding = 0;
+            //                         textarea.style.border = 'none';
+            //                         textarea.style.outline = 'none';
+            //                         textarea.style.boxShadow = 'none';
+            //                         textarea.style.background = 'transparent';
+            //                         textarea.value = entry.data;
+            //                         document.body.appendChild(textarea);
+            //                         textarea.select();
+            //                         try {
+            //                             var ret = document.execCommand('copy');
+            //                             var msg = ret ? 'successful' : 'unsuccessful';
+            //                             console.log('Copy text command was ' + msg);
+            //                         } catch (err) {
+            //                             console.log('Oops unable to copy');
+            //                         }
+            //                         document.body.removeChild(textarea);
+            //                     });
+            //             }
+            //         }
+            //     }
+            // });
+    
+            // var remoteListAddButton = row.find(".red-ui-editableList-addButton").hide();
+    
+            var hideGitUserEditForm = function() {
+                editGitUserButton.show();
+                formButtons.hide();
+                // $('.projects-dialog-remote-list-entry-delete').hide();
+                // remoteListAddButton.hide();
+    
+                gitUsernameLabel.show();
+                gitUsernameInput.hide();
+                gitEmailLabel.show();
+                gitEmailInput.hide();
+    
+            }
+    
+            $('<button class="editor-button">Cancel</button>')
+                .appendTo(formButtons)
+                .click(function(evt) {
+                    evt.preventDefault();
+                    hideGitUserEditForm();
+                });
+            
+            var saveButton = $('<button class="editor-button">Save</button>')
+                .appendTo(formButtons)
+                .click(function(evt) {
+                    evt.preventDefault();
+                    var spinner = addSpinnerOverlay(gitconfigContainer);
+    
+                    var body = {
```

Comment:
These properties should be stored in userSettings with a bit more structure:

```
{
    git: {
        user: {
            name: "name",
            email: "email"
        }
    }
}
```

as we don't know what else we will want to store in there in the future.